test/features/sshfs.feature

```diff
@@ -3,10 +3,19 @@ Feature: SSH Console
   (and vice versa), transient supports use sshfs to mount a host directory
   on the guest.
 
+  Scenario: Install SSHFS on a CentOS VM
+    Given a transient vm
+      And a disk image "centos/7:2004.01"
```

Comment:
This is a good solution for now. We should come back and change this to use a base image from https://github.com/ALSchwalm/transient-baseimages that (will eventually) have sshfs pre-installed and be available on the vagrant cloud